### ISSUE TITLE
Unable to create merged dataset emosaic error message

### ISSUE THREAD

**Issue by DavidT3** [DavidT3/XGA#159]
"createDatasetFailed raised by emosaic - The dataset with name '0108061801_0555780301_0827221401_0827220201_0604960201_0108062301_0827220101_0108060501_0827211101_0827220701_0604960401_0827220801_0827210901_0108060401_0108062101_0827221701_0555780101_0555780401_0108061901_0604960301_0604960101_0827220601_0555780201_0.5-2.0keVmerged_img.fits' could not be created. XCSRMY3-1 is the associated source. "

This is happening for some reason - first assumption would be something to do with the number of observations associated with this particular source. The source name in the error message is not currently correct, due to issue #158, but once I correct that bug I'll be able to track this to a particular source.

However, due to the number of observations I think this source is XCSRMY3-3977

**Comment by DavidT3** [comment]
I grabbed the command used to generate the merged image and ran it in command line, got the same error (as would be expected) - then changed the output name to something only a few characters long. The command then ran fine.

Appears I've bashed up against a limitation of SAS's emosaic command, this could be quite serious under rarified circumstances.

May have to rethink how I store XGA output products - consider doing a summary file of products for each directory.
